docs/how-to-guides/running-feast-in-production.md

```diff
@@ -141,6 +202,146 @@ prediction = model.predict(feature_vector)
+
+#### Load balancing
+
+The next step would be to install an L7 Load Balancer (eg, [Envoy](https://www.envoyproxy.io/)) in front of the Java feature server.
+For seamless integration with Kubernetes (including services created by Feast Helm chart) we recommend using [Istio](https://istio.io/) as Envoy's orchestrator.
+
+## 5. Ingesting features from a stream source
+
+Recently Feast added functionality for [Stream Ingestion](../reference/alpha-stream-ingestion.md).
+Please note that this is still in an early phase and new incompatible changes may be introduced.
+
+### 5.1. Using Python SDK in your Apache Spark / Beam pipeline
+
+The default option to write features from a stream is to add Python SDK into your existing PySpark / Beam pipeline.
+Feast SDK provides writer implementation that can be called from `foreachBatch` stream writer in PySpark like this:
+
+```python
+store = FeatureStore(...)
+
+def feast_writer(spark_df):
+    pandas_df = spark_df.to_pandas()
+    store.write_to_online_store("driver_hourly_stats", pandas_df)
+
+streamingDF.writeStream.foreachBatch(feast_writer).start()
+```
+
+### 5.2. Push service *(still under development)*
+
+Alternatively, if you want to ingest features directly from a broker (eg, Kafka or Kinesis) you can use "Push Service", which will write to Online Store on push call.
+This service will expose an HTTP API or when deployed on Serverless platforms like AWS Lambda or Google Cloud Function,
+this service can be directly connected to Kinesis or PubSub.
+
+If you are using Kafka, [HTTP Sink](https://docs.confluent.io/kafka-connect-http/current/overview.html) could be utilized as middleware.
+In this case "Push Service" can be deployed on Kubernetes or as a Serverless function.
+
+## 6. Monitoring
+
+Feast Services can report their metrics to a StatsD-compatible collector. To activate this function, you'll need to provide a StatsD IP address and a port when deploying the helm chart (in future will be added to `feature_store.yaml`).
+
+We use [InfluxDB-style extension](https://github.com/prometheus/statsd_exporter#tagging-extensions) for StatsD format to be able to send tags along with metrics. Keep that in mind while selecting the collector ([telegraph](https://www.influxdata.com/blog/getting-started-with-sending-statsd-metrics-to-telegraf-influxdb/#introducing-influx-statsd) will work for sure).
+
+We chose StatsD since it's de-facto a standard with various implementations (eg, [1](https://github.com/prometheus/statsd_exporter), [2](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/statsd/README.md))
+and metrics could be easily exported to Prometheus, InfluxDB, AWS CloudWatch, etc.
+
+---
+## Summary
+
+Summarizing it all together we want to show several options of architecture that will be most frequently used in production:
+
+### Option #1 (currently preferred)
+
+* Feast SDK is being triggered by CI (eg, Github Actions). It applies the latest changes from Repository to Feast Registry
```

Comment:
Nit: we typically don't capitalize things anymore unless it is referring to a class name.

```suggestion
* Feast SDK is being triggered by CI (eg, Github Actions). It applies the latest changes from the feature repo to the Feast registry
```